pkg/apis/ceph.rook.io/v1/types.go

```diff
@@ -499,7 +517,8 @@ type ClusterStatus struct {
 	CephVersion *ClusterVersion `json:"version,omitempty"`
 	// ObservedGeneration is the latest generation observed by the controller.
 	// +optional
-	ObservedGeneration int64 `json:"observedGeneration,omitempty"`
+	ObservedGeneration int64            `json:"observedGeneration,omitempty"`
+	Cephx              LocalCephxStatus `json:"cephx,omitempty"`
```

Comment:
We should use a custom status for CephCluster. Adding `csi` to the existing LocalCephxStatus will affect CephObjectStore's current implementation.